src/libraries/Common/tests/System/Security/Cryptography/X509Certificates/CertificateAuthority.cs

```diff
@@ -1035,7 +1038,7 @@ internal static KeyFactory RSASize(int keySize)
 
             internal static KeyFactory[] BuildVariantFactories()
             {
-                return [RSA, ECDsa];
+                return [RSA, ECDsa, MLDsa, SlhDsa];
```

Comment:
I expect this will need to change to something like

```suggestion
                List<KeyFactory> factories = [RSA, ECDsa];
                AddIf(factories, MLDsa, Cryptography.MLDsa.IsSupported);
                AddIf(factories, SlhDsa, Cryptography.SlhDsa.IsSupported);
                
                return factories.ToArray();
                
                static void AddIf(List<KeyFactory> factories, KeyFactory factory, bool condition)
                {
                    if (condition)
                    {
                        factories.Add(factory);
                    }
                }
```

But maybe there's some other magic I put in somewhere and don't remember to make it not need to be that hard.